emperor/support_files/js/abc-view-controller.js

```diff
@@ -183,5 +183,14 @@ define([
     throw Error('Not implemented');
   };
 
+  /**
+   * Writes the current settings to the active decomposition view(s).
+   * Will be called when the decomposition view is swapped out
+   * for a different view type.
```

Comment:
- Ray-casting is not working when looking at a paralle plot i.e. the lines won't reveal their name when you click on them.

Reply:
Added this.  It can't be pixel perfect: all calculations are done with a raycaster precision value in model coordinates, but if the user zooms in, they'll click the correct line every time.